crates/ruff_linter/src/rules/isort/snapshots/ruff_linter__rules__isort__tests__required_import_existing_import.py.snap

```diff
@@ -4,6 +4,6 @@ source: crates/ruff_linter/src/rules/isort/mod.rs
 I002 [*] Missing required import: `from __future__ import annotations`
 --> existing_import.py:1:1
 help: Insert required import: `from __future__ import annotations`
-1 + from __future__ import annotations
-2 | from __future__ import generator_stop
+1 | from __future__ import generator_stop
+2 + from __future__ import annotations
```

Comment:
Do you know why these snapshots changed?

Reply:
Before the fix, the required import was being inserted at the very beginning of the file, which would cause a syntax error since future imports must come first.

Now it's inserting the required import after the existing future import, which maintains the correct order and prevents the syntax error.